internal/controllers/utils/constants.go

```diff
@@ -92,3 +92,17 @@ const (
 	ClusterZtpDone    = "ZTP Done"
 	ClusterZtpNotDone = "ZTP Not Done"
 )
+
+// Hardware Provisioning status
+const (
+	HardwareProvisioning          = "In progress"
```

Comment:
Should it be `HardwareProvisioningInProgress`?

Reply:
Provisioning means “In process” :-)

Reply:
True! It was just for consistency i guess (name + what it's doing...pattern)

I guess the "best" way to deal with this would be have typed status 

```go
type HardwareProvisioningStatus string

InProgress  HardwareProvisioningStatus = "InProgress"
Failed      HardwareProvisioningStatus = "Failed"

etc etc 
```

But don't have any strong preference 

Btw totally missed it but should be "InProgress" (conditions types are expected to be PascalCase)

Reply:
Changed.